src/plugins/query_enhancements/public/plugin.tsx

```diff
@@ -101,6 +109,28 @@ export class QueryEnhancementsPlugin
       },
     });
 
+    data.__enhance({
+      ui: {
+        query: {
+          language: 'PPLAsync',
+          search: sqlAsyncSearchInterceptor,
```

Comment:
do we want ppl here?

Reply:
For now yes -- having an option-merge PR followup after this to finalize after PPL Async is working.

Reply:
Or maybe can make this PR do the option merging, in which case it'll be a while before it's out of draft.